ui/component/viewers/videoViewer/internal/videojs.jsx

```diff
@@ -247,19 +256,40 @@ export default React.memo<Props>(function VideoJs(props: Props) {
   useEffect(() => {
     const vjsElement = createVideoPlayerDOM(containerRef.current);
 
-    // Detect source file type via pre-fetch (async)
-    detectFileType().then(() => {
-      // Initialize Video.js
-      const vjsPlayer = initializeVideoPlayer(vjsElement);
+    // Initialize Video.js
+    const vjsPlayer = initializeVideoPlayer(vjsElement);
 
-      // Add reference to player to global scope
-      window.player = vjsPlayer;
+    // Add reference to player to global scope
+    window.player = vjsPlayer;
 
-      // Set reference in component state
-      playerRef.current = vjsPlayer;
+    // Set reference in component state
+    playerRef.current = vjsPlayer;
 
-      window.addEventListener('keydown', curried_function(playerRef, containerRef));
-    });
+    window.addEventListener('keydown', curried_function(playerRef, containerRef));
+
+    window.player.userActive(true);
+
+    if (autoplay) {
+      const videoPlayerDiv = document.querySelector('video');
+      if (videoPlayerDiv) {
+        videoPlayerDiv.click();
+      }
+      window.player.userActive(true);
+    }
```

Comment:
I believe this is to fix the issue of `video` not in focus, so pressing shortcuts like Down Arrow ends up scroll the page instead of adjusting the volume?

If yes, it doesn't seem to work for me.  Maybe remove for now?

Aside:
- Not a fan of DOM stuff unless no other choice.  Would `containerRef` (for the parent div) or `vjsElement` work?
- `videoPlayerDiv` -- "Div" confused me.  Is it asking for the parent div, or `<video>` tag itself?